cookbook/email/dev_environment.rst

```diff
@@ -140,11 +140,11 @@ by adding the ``delivery_whitelist`` option:
             delivery_whitelist:
                # all email addresses matching this regex will *not* be
                # redirected to dev@example.com
-               - "/@specialdomain.com$/"
+               - "/@specialdomain[.]com$/"
```

Comment:
If I am not mistaken, evaluating a character class is more expensive than evaluating a single character.